Clicker/ViewControllers/PollsDateViewController.swift

```diff
@@ -130,24 +131,42 @@ class PollsDateViewController: UIViewController {
         present(pollBuilderViewController, animated: true, completion: nil)
     }
     
+    func deleteSession(with id: Int) -> Future<DeleteResponse> {
+        return networking(Endpoint.deleteSession(with: id)).decode(DeleteResponse.self)
```

Comment:
same comment for all the functions which return Futures. i think you can just do `.decode()` if you have the type specified in the function declaration. wait on @AAAstorga  to confirm first tho